network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkImpl.java

```diff
@@ -1069,6 +1069,230 @@ public Ground getGround(String id) {
         return index.getGround(getIdFromAlias(id)).orElse(null);
     }
 
+    @Override
```

Comment:
I was more thinking about adding a section "// DC modelling". The same way we have "// VSC converter station".
And add the comment once for the section and not in all unimplemented method

Reply:
done